src/scp/NominationProtocol.h

```diff
@@ -122,5 +125,14 @@ class NominationProtocol
     // returns the latest message from a node
     // or nullptr if not found
     SCPEnvelope const* getLatestMessage(NodeID const& id) const;
+
+  private:
+    // The number of times the timer has expired
+    // Used for the quorum endpoint.
+    int32 mTimerExpCount;
+
+    // The number of times the timer has to expire before we consider the node
+    // missing. Used for reporting purposes only.
+    int32 const NUM_TIMEOUTS_THRESHOLD_FOR_REPORTING = 2;
```

Comment:
why do you have the same constant defined in two places? Just move it to "Slot"

Reply:
Moved!